src/compiler/checker.ts

```diff
@@ -2387,17 +2384,27 @@ namespace ts {
                     && node.parent.operatorToken.kind === SyntaxKind.EqualsToken
                     && isAliasableOrJsExpression(node.parent.right) ||
                 node.kind === SyntaxKind.ShorthandPropertyAssignment ||
-                node.kind === SyntaxKind.PropertyAssignment && isAliasableOrJsExpression((node as PropertyAssignment).initializer);
+                node.kind === SyntaxKind.PropertyAssignment && isAliasableOrJsExpression((node as PropertyAssignment).initializer) ||
+                isRequireVariableDeclaration(node, /*requireStringLiteralLikeArgument*/ true);

```

Comment:
(Not really relevant, but this thing made me look at the diff in Emacs, since it's incomprehensible...  Would look much better if the `||`s were at the BOLs, and `()` around the conjunctions...)